### ISSUE TITLE
Similar project

### ISSUE THREAD

**Issue by knownasilya** [EmberSherpa/ember-cli-deploy-gcloud#1]
I have a similar project, https://github.com/knownasilya/ember-cli-deploy-gcloud-storage

Let me know if you want to work together.

**Comment by taras** [comment]
Let's do it! I created my project specifically for uploading files to GCloud Storage to upload fastboot manifest file. I don't remember why I didn't go with your package. I think there was some specific piece missing.

Your package is more general for upload to GCloud Storage right?

**Comment by knownasilya** [comment]
Yes, basically it uploads everything ember-cli-deploy passes on at the moment, but you can specify files that you want to upload as well.

**Comment by taras** [comment]
@knownasilya It seems that the only difference between this library and your library is https://github.com/EmberSherpa/ember-cli-deploy-gcloud/blob/master/lib/tasks/upload.js#L28 and https://github.com/EmberSherpa/ember-cli-deploy-gcloud/blob/master/lib/tasks/upload.js#L39

These could be added to your library and I can deprecate mine.
